templates/taxi/driver_detail.html

```diff
@@ -1,18 +1,19 @@
 {% extends "base.html" %}
 
 {% block content %}
- <p><strong>
-    {{ driver.first_name }} {{ driver.last_name }}.</strong>
-    License number: {{ driver.license_number }}
-  </p>
-  {% for car in driver.cars.all %}
-    <hr>
-    <p><strong>Id:</strong>{{ car.id }}
-      <strong>Car model:</strong> {{ car.model }}
-    </p>
-
-  {% empty %}
-    <p>No cars!</p>
-  {% endfor %}
-
+  <h1 class="taxi-title">{{ driver.first_name }} {{ driver.last_name }}</h1>
+  {% if driver.cars %}
+    <h2 class="taxi-title">List of cars:</h2>
+    <ul class="car-list">
+      {% for car in driver.cars.all %}
```

Comment:
Verify that `driver.cars` is a related manager that supports the `.all()` method. If not, this could lead to errors in your application.